src/Components/CreateImageWizard/formComponents/Repositories.js

```diff
@@ -439,7 +441,9 @@ const Repositories = (props) => {
                               }}
                             />
                             <Td dataLabel={'Name'}>
-                              {repo.name}
+                              {repo.name
```

Comment:
nitpick: it's a bit confusing down the line where you have `repo.name ? repo.distribution : '-'` maybe you can add a new constant up top called `repoExists` and have that to true is the repo is set ?